docs/source/markdown/podman-create.1.md

```diff
@@ -364,6 +364,13 @@ The following example maps uids 0-2000 in the container to the uids 30000-31999
 
 Add additional groups to run as
 
+#### **--groups**=*false*
+
+Allow container to use the user's supplementary group access. If file systems or
```

Comment:
"Allow the container to retain the supplementary groups of the user launching Podman"